zuul.d/edpm_multinode.yaml

```diff
@@ -343,6 +343,15 @@
         dashboard_enabled: true
         cephfs_enabled: true
         ceph_nfs_enabled: true
+      # Override the Ceph container image and deploy Squid
```

Comment:
Do we really need the values here that are matching the defaults? Not really agains it but this means that we need to touch a few places to bump the version.

Reply:
mmm I wanted a place where we can have a well defined list of parameters that allow us to test the next version.
For this reason I can revert back the defaults under `defaults/main.yaml` to match `Reef`, and leave here, at job level, the overrides to install `Squid`.
How does it sounds?